translation/en/RedirectedResources/assets/abdata/adv/scenario/c24/25/84/translation.txt

```diff
@@ -1,11 +1,11 @@
 //
 // Dumped for Koikatsu v5.1 (CharaStudio) by KK_TextDump v1.4.6.0
 //
-「今更かもしれませんけど、コイカツ部ってお喋りをするだけの部活じゃないですよね？」=“I know it's a bit late to ask this, but the Koikatsu club isn't just a place for us to have chats, is it?”
+「今更かもしれませんけど、コイカツ部ってお喋りをするだけの部活じゃないですよね？」=“I know it's a bit late to ask this but, the Koikatsu club isn't just a place for us to have chats, is it?”
 「うん。何かしたいことがあったら提案してもらえると嬉しいな。その、無理のない範囲で……」=“Yeah. If there's anything you wanna do in particular, I'd be happy to go along with it. Uhm, as long as you keep it reasonable...”
-「うふふ♡　身構えなくても大丈夫ですよ♡　じゃあ……手を握ってもらっていいですか？」=“Ufufu♡ You don't need to on guard like that, you know♡ Well then... could we hold hands?”
+「うふふ♡　身構えなくても大丈夫ですよ♡　じゃあ……手を握ってもらっていいですか？」=“Ufufu♡ You don't need to prepare for a life-and-death battle like that, you know♡ Well then... could we hold hands?”
 「それぐらいなら……えっ？　手、すごく冷たいよ……」=“If it's just that much... eh--? Your hand, it's freezing cold...”
-「……あなたにとって『それぐらい』でも、私にとっては結構無理のある範囲なんですよ♡」=“...to you, it might be ‘just that much’, but I'm pushing myself quite a bit here, you know?♡”
-（よく見たら、震えてるじゃないか……）=(Now that I have a closer look, she's trembling...)
+「……あなたにとって『それぐらい』でも、私にとっては結構無理のある範囲なんですよ♡」=“...To you, it might be just that much" but, I'm pushing myself quite a bit here, you know?♡”
```

Comment:
```suggestion
「……あなたにとって『それぐらい』でも、私にとっては結構無理のある範囲なんですよ♡」=“...To you, it might be ‘just that much’, but I'm pushing myself quite a bit here, you know?♡”
```